pkg/kv/kvclient/rangefeed/rangefeed.go

```diff
@@ -228,6 +224,7 @@ func (f *RangeFeed) Start(ctx context.Context, spans []roachpb.Span) error {
 		defer pprof.SetGoroutineLabels(ctx)
 		ctx = pprof.WithLabels(ctx, pprof.Labels(append(f.extraPProfLabels, "rangefeed", f.name)...))
 		pprof.SetGoroutineLabels(ctx)
+		f.running.Add(1)
```

Comment:
Actually, we may have to do this, because the API contract says that it's invalid to call `Add()` on a zero-valued counter after calling `Wait()`.

https://pkg.go.dev/sync#WaitGroup.Add